android-bindings/lib-wrapper/android-bindings/publish.gradle

```diff
@@ -0,0 +1,52 @@
+apply plugin: 'maven-publish'
+
+version '1.2.0-pre0'
```

Comment:
The rest of the repo is now at `1.3.0-pre0'

Reply:
done